### ISSUE TITLE
Crash on command with no arguments

### ISSUE THREAD

**Issue by jaraco** [jaraco/irc#63]
Version: 12.4.2

Minimised test case:

``` python
import irc.client

reactor = irc.client.Reactor()
conn = irc.client.ServerConnection(reactor)
conn.connect("irc.twitch.tv", 6667, "")
conn._process_line("GLOBALUSERSTATE")
```

Backtrace:

```
Traceback (most recent call last):
  File "/home/andreas/irc-bug.py", line 6, in <module>
    conn._process_line("GLOBALUSERSTATE")
  File "/usr/lib64/python3.3/site-packages/irc/client.py", line 667, in _process_line
    target = arguments[0]
TypeError: 'NoneType' object is not subscriptable
```

---
- Bitbucket: https://bitbucket.org/jaraco/irc/issue/63
- Originally reported by: Andreas Ots
- Originally created at: 2015-07-17T11:03:16.219

**Comment by jaraco** [comment]
Add test capturing failure. Ref #63

→ <<cset df728886fcd9>>

---

Original comment by: Jason R. Coombs

**Comment by jaraco** [comment]
Always generate a list in Arguments.from_group. Set target to None if no arguments are available from which to draw a target. Fixes #63.

→ <<cset 55ae04d2b441>>

---

Original comment by: Jason R. Coombs

**Comment by jaraco** [comment]
Released as 12.4.3.

---

Original comment by: Jason R. Coombs
